### PR TITLE
[5.6] Added annotation for some facades for better type hinting

### PR DESCRIPTION
**Pull request by TBlindaruk**
https://github.com/laravel/framework/pull/23686 continues

### CODE REVIEW

File: src/Illuminate/Support/Facades/Log.php [hunk 1]
```diff
@@ -5,6 +5,16 @@
 use Psr\Log\LoggerInterface;
 
 /**
+ * @method static void emergency(string $message, array $context = array())
```
Comment (martinbean) [inline src/Illuminate/Support/Facades/Log.php]: Why is the long array syntax used here, but then the short array syntax used in other places, e.g. https://github.com/laravel/framework/pull/23694/files#diff-335842ccd61aadfa0c1ada80c98bb179R8?

Comment (TBlindaruk) [reply src/Illuminate/Support/Facades/Log.php]: @martinbean , 
I have copied from the Contracts.
I agree then it is not so good... but as for me will be better to save the original syntactic from the method.

Comment (a-komarev) [reply src/Illuminate/Support/Facades/Log.php]: It's better to change it to short array syntax `[]` because it breaks code styling.

Comment (TBlindaruk) [reply src/Illuminate/Support/Facades/Log.php]: @a-komarev , ok.
I will create a pull request later.

Comment (TBlindaruk) [reply src/Illuminate/Support/Facades/Log.php]: @a-komarev , @martinbean , https://github.com/laravel/framework/pull/23707

**Pull request merged**